### PR TITLE
netlist: Address comments from PR #9794, partially fix issue #9801

### PR DESCRIPTION
**Pull request by couriersud**
- added cstddef to includes in ppmf.h
- more explicit comments in ppmf.h
- Fixed more typos

### CODE REVIEW

File: src/lib/netlist/plib/plists.h [hunk 1]
```diff
@@ -39,7 +39,7 @@ namespace plib {
 		using reverse_iterator = std::reverse_iterator<iterator>;
 		using const_reverse_iterator = std::reverse_iterator<const_iterator>;
 
-		//uninitialised_array_t() noexcept = default;
+		//uninitialized_array_t() noexcept = default;
 		uninitialised_array() noexcept = default;
```
Comment (cuavas) [inline src/lib/netlist/plib/plists.h:43]: What’s the actual purpose of the comment here?  The comment uses US spelling while the code uses Commonwealth spelling for the class name.  It looks like a leftover from some kind of refactoring.

File: src/lib/netlist/macro/modules/nlmod_icl8038_dip.cpp [hunk 2]
```diff
@@ -52,7 +52,7 @@ NETLIST_START(ICL8038_DIP)
 
 	ALIAS(11, VI.ON) // GND
 	ALIAS(9, V_SQR.OP) // Square out
-	ALIAS(3, VO.OP) // Triag out
+	ALIAS(3, VO.OP) // Trigonal out
```
Comment (cuavas) [inline src/lib/netlist/macro/modules/nlmod_icl8038_dip.cpp:55]: Should be triangle, not “trigonal”.

**Review by cuavas — COMMENTED** [review]
There are a couple of issues with comments.  I’ll fix them myself to get this in sooner rather than later.

File: src/lib/netlist/plib/plists.h (hunk 1 above)
Comment (couriersud) [reply src/lib/netlist/plib/plists.h]: > It looks like a leftover from some kind of refactoring.

Yes, just delete the comment. What spelling is preferred? cspell should support both en-US and en-GB.

Comment (cuavas) [reply src/lib/netlist/plib/plists.h]: > What spelling is preferred?

I don’t care too much as long as it’s consistent across a unit.  Executive summaries for release announcements are written in Commonwealth English, but the summarised pull request descriptions preserve the author’s preference for US or Commonwealth spelling.

File: src/lib/netlist/macro/modules/nlmod_icl8038_dip.cpp (hunk 2 above)
Comment (couriersud) [reply src/lib/netlist/macro/modules/nlmod_icl8038_dip.cpp]: Yes, please change it. And everything else you like to please as well - I am not a native speaker and appreciate the feedback.

**Pull request merged**